src/main/java/gov/usds/case_issues/db/model/UserInformation.java

```diff
@@ -0,0 +1,50 @@
+package gov.usds.case_issues.db.model;
+
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.NaturalId;
+
+/**
+ * Metadata about the case issue data stored in this system
+ */
+@Entity
+public class UserInformation extends UpdatableEntity {
+
+	@NotNull
+	private Date lastSeen;
+	@NotNull
+	private String printName;
+	@NaturalId
+	@NotNull
+	private String userId;
+
+	protected UserInformation() {
+		/* for hibernate/JPA */ }
+
+	public UserInformation(String id, String printName) {
+		super();
+		updateLastSeen();
+		this.printName = printName;
+		this.userId = id;
+	}
+
+	public Date getLastSeen() {
+		return lastSeen;
+	}
+
+	public String getPrintName() {
+		return printName;
+	}
+
+	public String getId() {
+		return userId;
+	}
+
+	public void updateLastSeen() {
+		this.lastSeen = Date.from(ZonedDateTime.now().toInstant());
```

Comment:
I think this can actually just be `new Date()`. 😁 (Or did they deprecate that? It seems mean to deprecate it without adding a `now` static method like every other date/time library....)